h/static/scripts/group-forms/utils/api.ts

```diff
@@ -45,15 +45,24 @@ export class APIError extends Error {
 /* Make an API call and return the parsed JSON body or throw APIError. */
 export async function callAPI(
   url: string,
-  method: string = 'GET',
-  json: object | undefined,
+  {
+    method = 'GET',
+    json = null,
+    headers = {},
+  }: {
+    method?: string;
+    json?: object | null;
+    headers?: any;
+  } = {},
 ): Promise<object> {
+  headers['Content-Type'] = 'application/json; charset=UTF-8';
+
   const options: RequestInit = {
-    method: method,
-    headers: { 'Content-Type': 'application/json; charset=UTF-8' },
+    method,
+    headers,
   };
 
-  if (json !== undefined) {
+  if (json !== null) {
```

Comment:
Since `json` is `object | undefined` here, `if (json) { ... }` would also be fine. We can ignore all falsey values.

Reply:
Done: https://github.com/hypothesis/h/pull/8864/commits/aa982f96ac55791a33422f2a3c9d3ba94d75fe12